### ISSUE TITLE
Breaks on symbolic link

### ISSUE THREAD

**Issue by commandline** [balderdashy/include-all#12]
If include-all encounters a symbolic link, it fails gracelessly. I have a project that uses concurrently, this produces:
```
Error: ENOENT: no such file or directory, stat 'node_modules/.bin/concurrent'
      at Error (native)
      at Object.fs.statSync (fs.js:844:18)
      at node_modules/include-all/index.js:48:12
      at Array.forEach (native)
      at requireAll (node_modules/include-all/index.js:44:9)
      at node_modules/include-all/index.js:55:23
      at Array.forEach (native)
      at requireAll (node_modules/include-all/index.js:44:9)
      at buildDictionary (node_modules/sails/node_modules/sails-build-dictionary/index.js:68:14)
      at Function.module.exports.optional (node_modules/sails/node_modules/sails-build-dictionary/index.js:160:9)
      at Array.async.auto.nodeModulesFolder (node_modules/sails/lib/hooks/moduleloader/index.js:379:27)
      at node_modules/async/lib/async.js:484:38
      at _each (node_modules/async/lib/async.js:46:13)
      at Object.async.auto (node_modules/async/lib/async.js:455:9)
      at Hook.loadUserHooks (node_modules/sails/lib/hooks/moduleloader/index.js:363:13)
      at Hook.bound [as loadUserHooks] (node_modules/sails/node_modules/lodash/dist/lodash.js:729:21)
      at Hook.initialize (node_modules/sails/lib/hooks/userhooks/index.js:29:18)
      at Hook.bound [as initialize] (node_modules/sails/node_modules/lodash/dist/lodash.js:729:21)
      at node_modules/sails/lib/hooks/index.js:74:14
      at node_modules/async/lib/async.js:451:17
      at node_modules/async/lib/async.js:441:17
      at _each (node_modules/async/lib/async.js:46:13)
      at Immediate.taskComplete (node_modules/async/lib/async.js:440:13)
```
Changing line 48 to this resolves it:
```
    if (!fs.lstatSync(filepath).isSymbolicLink() && fs.statSync(filepath).isDirectory()) {
```
I will create a pull request with this change, too.

**Comment by sailsbot** [comment]
Thanks for posting, @commandline.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
